components/Categories.php

```diff
@@ -53,6 +53,12 @@ public function defineProperties()
                 'default'     => 'blog/category',
                 'group'       => 'winter.blog::lang.settings.group_links',
             ],
+            'categoryFilter' => [
+                'title'       => 'winter.blog::lang.settings.posts_filter',
+                'description' => 'winter.blog::lang.settings.posts_filter_description',
+                'default'     => '{{ :slug }}',
+                'type'        => 'string',
+            ],
```

Comment:
@RomainMazB three comments here:

- I think the name `categoryFilter` is probably not accurate for what we are doing. Instead, this should probably be `rootCategory`, as technically we are specifying a root category to start nesting from.
- I think those lang strings might be incorrect?
- Should the default perhaps be blank so that the original functionality is maintained?